File: ORB.Core/Models/CorporateRiskRiskMitigationAction.cs
using System;
using System.Collections.Generic;

namespace ORB.Core.Models
{
    public partial class CorporateRiskRiskMitigationAction : RiskRiskMitigationAction
    {
        public virtual CorporateRisk Risk { get; set; }
        public virtual CorporateRiskMitigationAction RiskMitigationAction { get; set; }
    }
}

File: corporate-reporting/src/types/CorporateRiskRiskMitigationAction.ts
import { ICorporateRiskMitigationAction } from "./CorporateRiskMitigationAction";
import { ICorporateRisk } from "./CorporateRisk";
import { IRiskRiskMitigationAction, RiskRiskMitigationAction } from "./RiskRiskMitigationAction";

export interface ICorporateRiskRiskMitigationAction extends IRiskRiskMitigationAction {
    Risk?: ICorporateRisk;
    RiskMitigationAction?: ICorporateRiskMitigationAction;
}

export class CorporateRiskRiskMitigationAction extends RiskRiskMitigationAction implements ICorporateRiskRiskMitigationAction { }